_posts/2015-02-06-Utilising-API-Blueprint-in-API-Clients.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Utilising API Blueprint in API Clients
+excerpt: Imagine an API client when you specify which resource and action you want to perform from an API Blueprint.
+layout: post
+date: 2015-02-06 18:00:00 +0000
+author: kyle
+published: true
+comments: true
+---
+
+I’m excited to release a new way to interface to API Blueprints. A client-side framework in Swift that allows you to introspect and perform actions at run-time.
+
+Imagine an API client when you specify which resource and action you want to perform from an API Blueprint, it can use the already defined information from a blueprint to build the request without having to duplicate the information such as the HTTP method and URI from your blueprint. This allows you to start building smarter clients which can evolve at run-time based on the blueprint.
```

Comment:
You could probably simplify this first sentence, as it was somewhat tautological:

"Imagine an API client that can use the already-defined information from an API Blueprint to build a request rather than hardcoding that information. This allows you to start building smarter clients which can evolve at run-time based on the blueprint."